platforms/fabric/src/main/resources/terra.accesswidener

```diff
@@ -1,3 +1,3 @@
 accessWidener v1 named
 
-extendable method net/minecraft/client/world/GeneratorType <init> (Ljava/lang/String;)V
```

Comment:
what changed lol

Reply:
the constructor was made `protected` so we can extend the class normally now

Reply:
what I'm saying is, it removed the line and then added the exact same one.
![image](https://user-images.githubusercontent.com/46940694/121122849-27dff800-c7f0-11eb-9703-5a1648f8e5e0.png)